react-app-rewired/headers/csps/defi/idle.ts

```diff
@@ -0,0 +1,17 @@
+import type { Csp } from '../../types'
+
+export const csp: Csp = {
+  'connect-src': [
+    'https://api.idle.finance',
+    'https://api-polygon.idle.finance',
+    'https://api.thegraph.com/subgraphs/name/samster91/idle-tranches',
```

Comment:
is this subgraph going to officially be supported by idle at some point? does idle themselves rely on this subgraph? just a little concerned about this being a personal account that may go away from a maintenance perspective.

Reply:
I can remove it for now